modules/creating-infra-machines.adoc

```diff
@@ -76,6 +82,8 @@ rendered-worker-daa08cc1e8f5fcdeba24de60cd955cc3            365c1cfd14de5b0e3b85
 ----
 
 . Optional: To deploy changes to a custom pool, create a machine config that uses the custom pool name as the label, such as `infra`. Note that this is not required and only shown for instructional purposes. In this manner, you can apply any custom configurations specific to only your infra nodes.
+
+.. Create a machine config:
```

Comment:
Do we need to ad a note about applying this causing a reboot?

Reply:
@sferich888 
PTAL at the change in my new PR: https://github.com/openshift/openshift-docs/pull/33057/commits/9069b2a88353049c6de646a10887068e72f00e9d